python_modules/libraries/dagster-embedded-elt/dagster_embedded_elt_tests/conftest.py

```diff
@@ -0,0 +1,38 @@
+import os
+import sqlite3
+import tempfile
+
+import pytest
+from dagster import file_relative_path
+from dagster_embedded_elt.sling import (
+    SlingResource,
+    SlingSourceConnection,
+    SlingTargetConnection,
+)
+
+
+@pytest.fixture
+def test_csv():
+    return os.path.abspath(file_relative_path(__file__, "test.csv"))
+
+
+@pytest.fixture
+def sqlite_connection(temp_db):
+    yield sqlite3.connect(temp_db)
+
+
+@pytest.fixture
+def temp_db():
```

Comment:
nit: use `tmp_path`: https://docs.pytest.org/en/7.1.x/how-to/tmp_path.html#the-tmp-path-fixture